tests/integration/states/pip.py

```diff
@@ -64,6 +64,16 @@ def test_pip_installed_errors(self):
                 shutil.rmtree(venv_dir)
 
```

Comment:
Oh we do? That would be swell. I briefly looked around a didn't see one, but I can reassess.